test/controllers/users_controller_test.rb

```diff
@@ -438,35 +438,53 @@ def change_user_password(user_factory, new_password)
         assert_match(/You do not have permission to perform this action./, flash[:alert])
       end
 
+      should "can give permissions to all applications even where signin is not delegatable or they don't have access to it" do
```

Comment:
Like `is forbidden for organisation admins even when assigning a user to organisations within their organisation subtree` below I decided that in this case we can also drop the suffix.